public/app/plugins/datasource/cloud-monitoring/types/types.ts

```diff
@@ -0,0 +1,62 @@
+import { DataQuery, SelectableValue } from '@grafana/data';
+import { DataSourceOptions, DataSourceSecureJsonData } from '@grafana/google-sdk';
+
+import { MetricKind } from './query';
+
+export interface CloudMonitoringVariableQuery extends DataQuery {
```

Comment:
some of this are also query-related so not sure if they should be in a different file. I would move everything to an index.ts file, with that you can avoid importing `.../types/types`, which is a bit weird.

Reply:
Agree it's a bit weird, this is a temporary solution until I can refactor out most of these types 😊